packages/experiments/src/components/Chiclet/ChicletCard.styles.ts

```diff
@@ -80,7 +80,7 @@ export const getStyles = (props: IChicletCardStyleProps): IChicletCardStyles =>
         color: palette.neutralPrimary,
         letterSpacing: 'normal',
         textAlign: 'left',
-        height: 41, // Two lines of text, making sure the third line is hidden
+        height: 37, // Two lines of text, making sure the third line is hidden
```

Comment:
kinda flaky values and worried that it might clip the text in some scenarios but perhaps let’s first surface those scenarios in usage.